### ISSUE TITLE
d3 as a bookmarklet / d3fx

### ISSUE THREAD

**Issue by nthitz** [visfest/topics2015#19]
So many people use d3 to create awesome websites but you can also use d3 to manipulate websites that are not yours and do some really wacky things. I'd also like to share a small d3 plugin I've created called [d3fx](https://github.com/nthitz/d3fx) to make fun animation automation even easier!

**Comment by enjalot** [comment]
+1 I'd be interested in more general discussion of bookmarklet/chrome extensions. there is also the [svg crowbar](http://nytimes.github.io/svg-crowbar/), [d3 deconstructor](https://chrome.google.com/webstore/detail/d3-deconstructor/papagkpjldglcaifeipgcfkhddlefnkk?hl=en) which allow us to transcend the browser sandbox.

**Comment by nthitz** [comment]
Agreed, I definitely want to touch on those as well as I've made both. Actually have crowbar in my notes of other bookmarklets to mention! :D

**Comment by jessihamel** [comment]
+1 D3fx is a great plugin. I'd love to see more.

**Comment by shobhitg** [comment]
+1 Very interested to see and explore more in bookmarklet world. We might also want to look into converting these into TamperMonkey and GreeseMonkey scripts.
